src/asm/parser.y

```diff
@@ -39,6 +39,7 @@
 uint32_t nListCountEmpty;
 int32_t nPCOffset;
 bool executeElseBlock; /* If this is set, ELIFs cannot be executed anymore */
+struct CaptureBody captureBody; /* Captures a REPT/FOR or MACRO */
```

Comment:
Shouldn't this be `static`?

Reply:
I don't think so; `nListCountEmpty`, `nPCOffset`, and `executeElseBlock` aren't. They're all used only within parser.y.

Reply:
Well, those should be `static` as well, though that change can be made separately if you prefer.

Reply:
No problem, they're `static` now.